.github/workflows/ci-build.yaml

```diff
@@ -101,6 +101,12 @@ jobs:
             make MELANGE="melange" MELANGE_EXTRA_OPTS="--create-build-log --cache-dir=$(pwd)/../.melangecache" REPO="$GITHUB_WORKSPACE/packages" package/$package -j1
           done
 
+      - name: "Check that packages can be installed with apk add"
+        run: |
+          for f in $(find packages -name '*.apk'); do
+            apk add --allow-untrusted $f
```

Comment:
We want a --simulate to avoid conflicting packages failing.

Reply:
Also we might want to overwrite etc/apk/world and lib/apk/db/installed with empty files for the same reason.